bundles/org.openhab.binding.myq/README.md

```diff
@@ -0,0 +1,71 @@
+# MyQ Binding
+
+This binding integrates with the [The Chamberlain Group MyQ](https://www.myq.com) cloud service. It allows monitoring and control over [MyQ](https://www.myq.com) enabled garage doors manufactured by LiftMaster, Chamberlain and Craftsman.
+
+## Supported Things
+
+### Account
+
+This represents the MyQ cloud account and uses the same credentials needed when using the MyQ mobile application.
+
+ThingTypeUID:`account`
+
+### Garage Door
+
+This represents a garage door associated with an account.  Multiple garage doors are supported.
+
+ThingTypeUID:`garagedoor`
+
+### Lamp
+
+This represents a lamp associated with an account.  Multiple lamps are supported.
+
+ThingTypeUID:`lamp`
+
+## Discovery
+
+Once an account has been added, garage doors and lamps will automatically be discovered and added to the inbox.
+
+## Channels
+
+| Channel       | Item Type     | Thing Type       | Description              |
+|---------------|---------------|------------------|--------------------------|
+| status        | String        | garagedoor, lamp | Status of the device     |
```

Comment:
Can you mention which state the user can expect?

Reply:
There are state descriptions in the XML as well.

Reply:
Yeah, but do you want to ask the user to read the source? Better document it here.

Reply:
fair enough